src/tpm2-tss-engine-common.c

```diff
@@ -209,12 +209,11 @@ tpm2tss_tpm2data_readtpm(uint32_t handle, TPM2_DATA **tpm2Datap)
     ESYS_CONTEXT *esys_ctx = NULL;
     TPM2B_PUBLIC *outPublic;
 
-    tpm2Data = OPENSSL_malloc(sizeof(*tpm2Data));
+    tpm2Data = OPENSSL_zalloc(sizeof(*tpm2Data));
```

Comment:
Openssl 1_0_2g does not have this... on top of that, thier zalloc is literally just malloc + memset anyways. dropping